lib/internal/event_target.js

```diff
@@ -286,6 +290,17 @@ class EventTarget {
     }
     type = String(type);
 
+    if (signal) {
+      if (signal.aborted) {
```

Comment:
@addaleax - can you please confirm it's ok (conceptually) that it's fine we don't call `kNewListener` and `kRemoveListener` in this case? I think it should be fine because workers only use this for reference counting but I want to make sure there aren't additional cases I haven't considered.

Reply:
Yeah, short-circuiting like this this should be okay as far as workers are concerned :+1: